move/sources/canvas/token.move

```diff
@@ -602,6 +616,17 @@ module addr::canvas_token {
         simple_set::remove(&mut canvas_.blocklisted_artists, &addr);
     }
 
+    public entry fun update_max_number_of_piexls_per_draw(
+        caller: &signer,
+        canvas: Object<Canvas>,
+        updated_max_number_of_pixels_per_draw: u64,
+    ) acquires Canvas {
+        let caller_addr = signer::address_of(caller);
+        assert_is_admin(canvas, caller_addr);
+        let canvas_ = borrow_global_mut<Canvas>(object::object_address(&canvas));
```

Comment:
is this `let canvas_` a known convention? feels a bit weird

Reply:
it is in this repo